homeassistant/components/system_log.py

```diff
@@ -0,0 +1,121 @@
+"""
+Support for system log.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/system_log/
+"""
+import os
+import re
+import asyncio
+import traceback
+from io import StringIO
+
+import voluptuous as vol
+
+from homeassistant.bootstrap import DATA_SYSTEM_LOG
+from homeassistant.config import load_yaml_config_file
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.http import HomeAssistantView
+
+DOMAIN = 'system_log'
+DEPENDENCIES = ['http']
+SERVICE_CLEAR = 'clear'
+
+CONF_MAX_ENTRIES = 'max_entries'
+
+DEFAULT_MAX_ENTRIES = 50
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Optional(CONF_MAX_ENTRIES,
+                     default=DEFAULT_MAX_ENTRIES): cv.positive_int,
+    }),
+}, extra=vol.ALLOW_EXTRA)
+
+SERVICE_CLEAR_SCHEMA = vol.Schema({})
+
+
+@asyncio.coroutine
+def async_setup(hass, config):
+    """Set up the logger component."""
+    conf = config[DOMAIN]
+
+    handler = hass.data[DATA_SYSTEM_LOG]
+    handler.resize(conf.get(CONF_MAX_ENTRIES))
+
+    hass.http.register_view(AllErrorsView(handler))
+    yield from hass.components.frontend.async_register_built_in_panel(
+        'system-log', 'System Log', 'mdi:monitor')
```

Comment:
To be my own advocate, you can replace `'System Log'` here with `'system_log'`, then add an entry to the translations:
https://github.com/home-assistant/home-assistant-polymer/blob/1f9fc46576abfb5a644e86930f85698071dcdec5/src/translations/en.json#L2

I think this is the first new panel since translations started, so it would be a good test to make sure the workflow is reasonable.

Reply:
Ah, yes. I was thinking about how that worked. Thanks for the clarification, I'll fix that!